### PR TITLE
Delete SQLQuery and change it to promise

### PR DESCRIPTION
**Pull request by JLiao98**

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -1,19 +1,32 @@
 const express = require('express');
-const { SQL } = require('../sql.js');
+const action = require('../actions.js');
 
 const router = express.Router();
 
-router.get('/device', async (req, res) => {
-  const query = 'SELECT device_id FROM public.device;';
+router.get('/device', (req, res) => {
+  action
```
Comment (awave1) [inline routes/users.js]: you can do
```js
try {
  const data = await actions.getDevices();
  res.send(data);
} catch (err) {
  // Handle error here
}
```

Comment (JLiao98) [reply routes/users.js]: Oh but I'll have to write another async function. Using .then() and .catch() are the same I guess?

Comment (awave1) [reply routes/users.js]: You can just make
```js
 	router.get('/device', async (req, res) => { ... }
```

Comment (JLiao98) [reply routes/users.js]: Yeah I put that in action.js

**Pull request merged**